### PR TITLE
Jdi

### PR DESCRIPTION
**Pull request by Le0nRoy**

### CODE REVIEW

File: src/test/java/hw/jdi/MetalsAndColorsTestData.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
+package hw.jdi;
+
+import java.util.List;
+
+public class MetalsAndColorsTestData {
```
Comment (troe88) [inline src/test/java/hw/jdi/MetalsAndColorsTestData.java:5]: Could you please follow Java Code Convention, most all of your code have to be reformatted.

File: src/test/java/hw/jdi/MetalsAndColorsTestData.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    private List<Integer> summary;
+    private List<String> elements;
+    private String color;
+    private String metals;
+    private List<String> vegetables;
+
+    public MetalsAndColorsTestData(List<Integer> summary, List<String> elements, String color, String metal, List<String> vegetables) {
```
Comment (troe88) [inline src/test/java/hw/jdi/MetalsAndColorsTestData.java:13]: Why don't use use Lombok here ?

File: src/test/java/hw/jdi/BaseClass.java [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+        metalsAndColorsPage.builder()
+                .setSummaryValues(testData.getSummary())
+                .setElementsValues(testData.getElements())
+                .setColorsValue(testData.getColor())
+                .setMetalsValue(testData.getMetals())
+                .setVegetablesValue(testData.getVegetables())
+                .submit();
+
+        List<String> resultsPattern = new ArrayList<String>();
```
[the remark this answers is not in the document]
Comment (Le0nRoy) [reply src/test/java/hw/jdi/BaseClass.java]: Did not understand what exactly and how to encapsulate.

**Comment by Le0nRoy** [comment]
Now must be better

**Pull request merged**